csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T].cs

```diff
@@ -6,36 +6,23 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Provides the base class from which the classes that represent all segments walker are derived. This is an abstract class.</para>
+    /// <para>Предоставляет базовый класс, от которого наследуются классы, реализующие AllSegmentsWalkerBase. Это абстрактный класс.</para>
     /// </summary>
     /// <seealso cref="AllSegmentsWalkerBase{T, Segment{T}}"/>
     public abstract class AllSegmentsWalkerBase<T> : AllSegmentsWalkerBase<T, Segment<T>>
     {
         /// <summary>
-        /// <para>
-        /// Creates the segment using the specified elements.
-        /// </para>
-        /// <para></para>
+        /// <para>Initializes a new instance of the <see cref="Segment"/> class, using the <paramref name="base"/>,<paramref name="offset"/> and <paramref name="length" />.</para>
+        /// <para>Инициализирует новый экземпляр класса <see cref="Segment"/>, используя <paramref name="base"/>,<paramref name="offset"/> и <paramref name="length"/> </para>
         /// </summary>
-        /// <param name="elements">
-        /// <para>The elements.</para>
-        /// <para></para>
-        /// </param>
-        /// <param name="offset">
-        /// <para>The offset.</para>
-        /// <para></para>
-        /// </param>
-        /// <param name="length">
-        /// <para>The length.</para>
-        /// <para></para>
-        /// </param>
+        /// <param name="base"><para>The reference to the original list containing the elements of this segment.</para><para>Ссылка на исходный список в котором находятся элементы этого сегмента.</para></param>
+        /// <param name="offset"><para>The offset relative to the <paramref name="base"/> list from which the segment starts.</para><para>Смещение относительно списка <paramref name="base"/>, с которого начинается сегмент.</para></param>
+        /// <param name="length"><para>The segment's length.</para><para>Длина сегмента.</para></param>
         /// <returns>
-        /// <para>A segment of t</para>
-        /// <para></para>
-        /// </returns>
+        /// <para>Instance of the Segment class<T></para>
+        /// <para>Экземпляр класса Segment</para>
```

Comment:
Use more full comments

Reply:
Use a `see` tag for the `Segment`, by the way the `see` tag can refer to Generic class, therefore do not remove <T>, use `{T}` in the `see` tag